react/features/base/conference/actions.js

```diff
@@ -757,10 +759,6 @@ export function setSubject(subject: string = '') {
         const { conference } = getState()['features/base/conference'];
 
         if (conference) {
-            dispatch({
-                type: SET_PENDING_SUBJECT_CHANGE,
```

Comment:
What was this event originally?

Reply:
it clears the "pending" subject, I moved that logic to the middleware, so it happens when the subject was updated